Xcode-Configurations/Base.xcconfig

```diff
@@ -46,3 +46,7 @@ GCC_WARN_UNINITIALIZED_AUTOS = YES
 GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE
 GCC_WARN_UNUSED_FUNCTION = YES
 GCC_WARN_UNUSED_VARIABLE = YES
+
+GITUP_PLATFORM[sdk=macosx*] = MacOSX
+GITUP_PLATFORM[sdk=iphonesimulator*] = iPhoneSimulator
+GITUP_PLATFORM[sdk=iphoneos*] = iPhoneOS
```

Comment:
The only thing I’d like to add is a comment above the first declaration explaining why this change is necessary. Helps if someone has to look at it later and wonder what it’s for.